rest/config_test.go

```diff
@@ -2529,3 +2529,85 @@ func TestBucketCredentialsValidation(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
I think it would be a good idea to start putting the serverless tests in their own file(s), so that we can maintain a better picture of the suite of tests we have for serverless.